cli/src/torii/routing.rs

```diff
@@ -99,7 +99,7 @@ pub(crate) async fn handle_instructions(
     sumeragi: Arc<Sumeragi>,
     transaction: VersionedTransaction,
 ) -> Result<Empty> {
-    let wsv = sumeragi.get_clone_of_world_state_view(); // This probably means this function doesn't work
+    let wsv = sumeragi.get_clone_of_world_state_view();
```

Comment:
Does this definitely mean that the function definitely works?

Reply:
Yes. Because the function is only reading and the broadcast queue's and arc's get cloned properly as well. If this function didn't work then we wouldn't have 90+ % pass rate on the integration tests.